cmd/syncer/options/options.go

```diff
@@ -52,7 +54,7 @@ func (options *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&options.PclusterID, "workload-cluster-name", options.PclusterID,
 		fmt.Sprintf("ID of the -to cluster. Resources with this ID set in the '%s' label will be synced.", nscontroller.ClusterLabel))
 	fs.StringArrayVarP(&options.SyncedResourceTypes, "sync-resources", "r", options.SyncedResourceTypes, "Resources to be synchronized in kcp.")
-
+	fs.StringVar(&options.KCPHost, "kcp-host", options.KCPHost, "Hostname of the kcp server (host:port). If not set, defaults to the --from-kubeconfig cluster URL.")
```

Comment:
Credits to @aojea for the question: do we know why we need this flag?

My (maybe naive) assumption: every erver URL good for the syncer in the provided `--from-kubeconfig` is good enough for every other pods.